sphinx/themes/basic/static/searchtools.js

```diff
@@ -164,7 +164,7 @@ const Search = {
     const htmlElement = new DOMParser().parseFromString(htmlString, 'text/html');
     htmlElement.querySelectorAll(".headerlink").forEach((el) => { el.remove() });
     const docContent = htmlElement.querySelector('[role="main"]');
-    if (docContent !== undefined) return docContent.textContent;
+    if (docContent) return docContent.textContent;
```

Comment:
This looks good to me, and seems consistent with our validation of `querySelector` results in `doctools.js` here: https://github.com/sphinx-doc/sphinx/blob/80d5396f9d434d7c71d6f7ac5fa727a972b69584/sphinx/themes/basic/static/doctools.js#L124-L125